sportsdataverse/_common_espn_player_stats.py

```
@@ -0,0 +1,334 @@
+    if return_as_pandas:
+        return frame.to_pandas()
+    return frame
+
+
+def _safe_json(url: str | None, **kwargs: Any) -> dict[str, Any] | None:
+    """Best-effort GET + ``.json()``; returns ``None`` on any failure."""
+    if not url:
+        return None
+    try:
+        return download(url, **kwargs).json()
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)